project/README.rst

```diff
@@ -28,29 +28,23 @@ entry too.  You can request my review in the PR.
 
 .. Presentation
 .. ============
-
```

Comment:
Please do not make the unnecessary changes to the blank lines.  Leave a message after fixing it.